go.mod

```diff
@@ -330,3 +330,5 @@ require (
 )
 
 replace golang.org/x/vuln => github.com/luhring/golang-vuln v1.0.2-0.20231029212121-c364fd4725dc
+
+replace github.com/github/go-spdx/v2 v2.2.0 => github.com/darsh12/go-spdx/v2 v2.0.0-20240506204000-9af17522cc6e
```

Comment:
Is this fork proposed anywhere as a PR on the main repo? If not, is there an issue describing why it wouldn't be?

I'm a little nervous about relying on an unknown maintainer for this. Not because of vulns (it's pinned and looks sane), but more for maintenance burden going forward. If go-spdx adds features, will this fork integrate them?

Reply:
@imjasonh it is proposed here https://github.com/github/go-spdx/pull/57 i don't know how else we can pull it.

But basically there are a couple of licenses that we use; and are approved; but unknown. The alternative is for me to add them to exceptions like I did for custom in the other pr.

It is like cryptsetup WITH cryptsetup-OpenSSL exception and Boehm-GC.

Reply:
Thanks that's helpful!